src/gui/spaces/spacesdelegate.cpp

```diff
@@ -94,37 +123,50 @@ QSize SpacesDelegate::sizeHint(const QStyleOptionViewItem &option, const QModelI
         auto opt = openBrowserButtonRect(option);
         return opt.rect.size();
     }
+
     return QStyledItemDelegate::sizeHint(option, index);
 }
 
 bool SpacesDelegate::editorEvent(QEvent *event, QAbstractItemModel *model, const QStyleOptionViewItem &option, const QModelIndex &index)
 {
     if (index.column() == static_cast<int>(SpacesModel::Columns::WebUrl)) {
+        // handle click on button
+        // TODO: simulate "click on button" visually, too (i.e., render button accordingly)
```

Comment:
Probably not needed, on 99% of the systems the browser will appear super fast.

Reply:
It's in there mostly as a reminder. We can still ignore it every time we stumble across it. But we might want to change the current behavior at some point. At least it documents the intention, i.e., "it's okay for now, but it could be improved".